client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesRequestConverters.java

```diff
@@ -137,6 +137,7 @@ static Request closeIndex(CloseIndexRequest closeIndexRequest) {
         parameters.withTimeout(closeIndexRequest.timeout());
         parameters.withMasterTimeout(closeIndexRequest.masterNodeTimeout());
         parameters.withIndicesOptions(closeIndexRequest.indicesOptions());
+        parameters.withWaitForActiveShards(closeIndexRequest.waitForActiveShards());
```

Comment:
This is actually a bugfix, we didn't respect this parameter before this change.

Reply:
Nice catch.